std-bits/table/src/main/resources/META-INF/native-image/org/enso/table/jni-config.json

```diff
@@ -0,0 +1,29 @@
+[
+  {
+    "name":"java.awt.Color",
+    "fields":[{"name":"value"}]
+  },
+  {
+    "name":"java.awt.GraphicsEnvironment",
```

Comment:
It would be better to create substitutions for these methods. Something like
```java
@TargetClass(java.awt.GraphicsEnvironment.class)
class Target_GraphicsEnvironment {
   @Substitution
   public boolean isHeadless() { return true; }
}
```
rather than enabling `java.desktop`.